typing_extensions/src/test_typing_extensions.py

```diff
@@ -2230,6 +2230,52 @@ def test_no_isinstance(self):
             issubclass(int, TypeGuard)
 
 
+class LiteralStringTests(BaseTestCase):
+    def test_basics(self):
+        class Foo:
+            def bar(self) -> LiteralString: ...
+
+        self.assertEqual(gth(Foo.bar), {'return': LiteralString})
```

Comment:
```suggestion
        self.assertEqual(gth(Foo.bar), {'return': LiteralString})
        self.assertIsNone(get_origin(LiteralString), None))
```

Reply:
Oups, `@skipUnless(PEP_560, "Python 3.7 required")` is needed!

Reply:
And your suggestion had a syntax error :D